modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,119 @@
++
+.Example 
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+ name: argocd-sample
+ namespace: default
+spec:
+ monitoring:
+   disableMetrics: true
+----
++
+Use the following command to scrape metrics in the Argo CD CLI:
++
+[source,terminal]
+----
+$ oc patch argocd argocd-sample -n default --type='json' -p='[{"op": "replace", "path": "/spec/monitoring/disableMetrics", "value": true}]'
+----
+
+* With this update, the Argo Rollouts functionality in {gitops-title} is promoted from Technology Preview (TP) to General Availability (GA). Use this feature to perform the following actions: link:https://issues.redhat.com/browse/GITOPS-3848[GITOPS-3848]
+
+** Run cluster-scoped Argo Rollouts instead of namespace-scoped
+** Route traffic by using Openshift Routes for canary application deployment 
+** Configure Argo Rollouts CLI
+
+* With this update, you can perform canary-based Argo Rollouts deployments by using OpenShift Routes-based traffic management. Argo Rollouts supports traffic splitting by modifying the percentage of traffic that reaches the canary or stable application version during a deployment. link:https://issues.redhat.com/browse/GITOPS-2400[GITOPS-2400]
+
+* With this update, the Argo CD Applications in any namespace feature is promoted from Technology Preview (TP) to General Availability (GA). link:https://issues.redhat.com/browse/GITOPS-3796[GITOPS-3796]
+
+* With this update, the following new commands are introduced for the Apps in any namespace feature:
+** `argocd proj add-source-namespace` - Use this command to add source namespaces to `AppProject.Spec.SourceNamespaces`, for whitelisting specific namespaces for application creation.
+** `argocd proj delete-source-namespace` - Use this command to remove source namespaces from `AppProject.Spec.SourceNamespaces`, and for blacklisting specific namespaces. link:https://issues.redhat.com/browse/GITOPS-4045[GITOPS-4045]
+
+* With this update, you can customize permissions for a cluster-scoped Argo CD instance by creating user-defined `ClusterRoles` and `ClusterRoleBindings` for the Argo CD Application Controller and server components. link:https://issues.redhat.com/browse/GITOPS-2614[GITOPS-2614]
+
+* With this update, you can customize the Keycloak route name in the {gitops-title} Operator. This update also enables support for complex routing scenarios, such as dividing routes among various ingress controller shards. link:https://issues.redhat.com/browse/GITOPS-4451[GITOPS-4451]
+
+* With this update, the default TLS termination policy of the Argo CD server's Route changed from `Passthrough` to `Reencrypt`. As a result, the TLS connection now relies on the default ingress operator certificate instead of the self-signed certificate generated by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-3918[GITOPS-3918]
```

Comment:
```suggestion
* With this update, the default TLS termination policy of the Argo CD server's Route changed from `Passthrough` to `Reencrypt`. As a result, the TLS connection relies on the default ingress Operator certificate instead of the self-signed certificate generated by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-3918[GITOPS-3918]
```
https://redhat-documentation.github.io/supplementary-style-guide/#operator

Reply:
Fixed.